bin/build-ember

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+
+rm -rf build/ember-app
+cd src/ember-app
+./node_modules/ember-cli/bin/ember build --output-path ../../build/ember-app/
```

Comment:
Yes, you can turn finger printing off to disable that behavior.  More details here: https://ember-cli.com/user-guide/#fingerprinting-and-cdn-urls